src/Module/CommandCommonOptions.php

```diff
@@ -48,12 +48,18 @@ public static function addDefaultOptionsAndArguments(Command $command) : void
     }
 
     /**
-     * Retrieve the modules path from input
+     * Retrieve the modules path from  1: $input, 2: project config or 3: default 'src'
+     *
+     * @param InputInterface $input
+     * @param array $config
+     * @return string
      */
-    public static function getModulesPath(InputInterface $input) : string
+    public static function getModulesPath(InputInterface $input, array $config = []) : string
     {
-        $modulesPath = $input->getOption('modules-path') ?: 'src';
-        $modulesPath = preg_replace('/^\.\//', '', str_replace('\\', '/', $modulesPath));
+        $configuredModulesPath = $config[self::class]['modules_path'] ?? null;
+        $modulesPath           = $input->getOption('modules-path') ?? $configuredModulesPath ?: 'src';
```

Comment:
I'm going to change line 59 to:

```php
$configuredModulesPath = $config[self::class]['modules_path'] ?? 'src';
```

This will eliminate the need for the nested ternary in the following line, which can then become:

```php
$modulesPath = $input->getOption('modules-path') ?? $configuredModulesPath;
```